test/e2e/plain_provisioner_test.go

```diff
@@ -1337,7 +1337,7 @@ var _ = Describe("plain provisioner bundle", func() {
 					WithTransform(func(c *metav1.Condition) string { return c.Type }, Equal(rukpakv1alpha2.TypeInstalled)),
 					WithTransform(func(c *metav1.Condition) metav1.ConditionStatus { return c.Status }, Equal(metav1.ConditionFalse)),
 					WithTransform(func(c *metav1.Condition) string { return c.Reason }, Equal(rukpakv1alpha2.ReasonInstallFailed)),
-					WithTransform(func(c *metav1.Condition) string { return c.Message }, ContainSubstring(`the server could not find the requested resource`)),
+					WithTransform(func(c *metav1.Condition) string { return c.Message }, ContainSubstring(`no matches for kind "CatalogSource" in version "operators.coreos.com/v1alpha1"`)),
```

Comment:
Though `no matches for the kind` and `could not find the resources` return the same HTTP error code, I believe the former means that the CRD is not installed available in the cluster (API server does not know the definition), whereas the latter indicates that the specific resource endpoint does not exist. Was there a change in k8s because of which this change was needed? Just curious.